src/textarea/textarea.tsx

```diff
@@ -9,25 +10,120 @@ interface Props {
     className?: string;
     placeholder?: string;
     disabled?: boolean;
+    enableCharacterLimit?: boolean;
 }
 
-const Textarea: React.FunctionComponent<Props> = (props) => {
-    const onChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => props.onChange(event.target.value);
-    return (
-        <div className="textarea-component">
-            <textarea
-                id={props.id}
-                className={`form-control ${props.className || ''}`}
-                rows={props.rows || 3}
-                onBlur={props.onBlur}
-                onChange={onChange}
-                placeholder={props.placeholder}
-                data-qa={`textarea--element-${props.id}`}
-                value={props.value}
-                disabled={props.disabled}
-            />
-        </div>
-    );
-};
+interface State {
+    rows: number;
+    value: string;
+    limit: number;
+    stroke: string;
+    strokeDasharray: string;
+    outputText: string;
+    radius: number;
+}
+
+enum TextareaConfig {
```

Comment:
Interesting use of an enum, I personally would prefer just a set of constants. Please use existing naming patterns: `minRows` => `MIN_ROWS`

Reply:
i too changed it to constant